core/vcsl/vcsl_geocentric.h

```diff
@@ -25,14 +25,14 @@
 class vcsl_geocentric
   : public vcsl_geographic
 {
+ public:
   //***************************************************************************
   // Constructors/Destructor
   //***************************************************************************
 
   // Default constructor.
   vcsl_geocentric();
```

Comment:
This constructor is now public. Was this intended? Similar thing an a few other files.

Reply:
Fix.  Now explicitly private (instead of default to private).

Reply:
I think there were a few more. Did you update them too?

Reply:
There are 3.    I don't understand the point of this code. 

- If some constructor is private, it means that no one but the class itself (and friends) should be able to create instances of it using that constructor. Therefore, you can provide static methods like getInstance() to create instances of the class or create the instances in some friend class/method.

There is no static or friend class/method.  
 Very confusing, but the code is back to the way it used to be for.  I'll leave it to someone else to fix. this code that I don't need.

Reply:
I suspected so. In that case, those constructors could be publicly deleted?